docker-compose/setup-script/noco.sh

```diff
@@ -477,37 +532,143 @@ server {
+    echo "4. Restart"
+    echo "5. Upgrade"
+    echo "6. Scale"
+    echo "7. Monitoring"
+    echo "0. Exit"
+}
+
+# Function to start the service
+start_service() {
+    echo -e "\nStarting nocodb..."
+    $DOCKER_COMMAND compose up -d
+}
+
+# Function to stop the service
+stop_service() {
+    echo -e "\nStopping nocodb..."
+    $DOCKER_COMMAND compose stop
+}
+
+# Function to show logs
+show_logs() {
+    clear
+    echo "Select a container for logs:"
+    echo "1. nocodb"
+    echo "2. db"
+    echo "3. nginx"
+    echo "4. redis"
+    echo "5. watchtower"
+    echo "6. All"
+    echo "0. Back to main menu"
+    echo "Enter your choice: "
+
+    read -n 1 log_choise
+
+    case \$log_choise in
+        1) $DOCKER_COMMAND compose logs -f nocodb ;;
+        2) $DOCKER_COMMAND compose logs -f db ;;
+        3) $DOCKER_COMMAND compose logs -f nginx ;;
+        4) $DOCKER_COMMAND compose logs -f redis ;;
+        5) $DOCKER_COMMAND compose logs -f watchtower ;;
+        6) $DOCKER_COMMAND compose logs -f ;;
+        0) return ;;
+        *) echo "Invalid choice. Returning to main menu." ;;
+    esac
+}
+
+# Function to restart the service
+restart_service() {
+    echo -e "\nRestarting nocodb..."
+    $DOCKER_COMMAND compose restart
+}
+
+# Function to upgrade the service
+upgrade_service() {
+    echo -e "\nUpgrading nocodb..."
+    $DOCKER_COMMAND compose pull
+    $DOCKER_COMMAND compose up -d --force-recreate
+    $DOCKER_COMMAND image prune -a -f
+}
+
+# Function to scale the service
+scale_service() {
+    num_cores=\$(nproc)
+    current_scale=\$($DOCKER_COMMAND compose ps -q nocodb | wc -l)
+    echo -e "\nCurrent number of instances: \$current_scale"
+    echo "How many instances of NocoDB do you want to run (Maximum: \${num_cores}) ? (default: 1): "
+    scale_num=\$(read_number_range 1 \$num_cores)
+
+    if [ \$scale_num -eq \$current_scale ]; then
+        echo "Number of instances is already set to \$scale_num. Returning to main menu."
+        return
+    fi
+
+    $DOCKER_COMMAND compose up -d --scale nocodb=\$scale_num
+}
+
+# Function for basic monitoring
+monitoring_service() {
+    echo -e '\nLoading stats...'
+    $DOCKER_COMMAND stats
+}
+
+# Main program loop
+while true; do
+    show_menu
+    echo "Enter your choice: "
+
+    read -n 1 choice
+    case \$choice in
+        1) start_service && MSG="NocoDB Started" ;;
+        2) stop_service && MSG="NocoDB Stopped" ;;
+        3) show_logs ;;
+        4) restart_service && MSG="NocoDB Restarted" ;;
+        5) upgrade_service && MSG="NocoDB has been upgraded to latest version" ;;
+        6) scale_service && MSG="NocoDB has been scaled" ;;
+        7) monitoring_service ;;
+        0) exit 0 ;;
+        *) MSG="\nInvalid choice. Please select a correct option." ;;
+    esac
+done
 EOF
-fi
 
+message_arr+=("Help script: help.sh")
 
-message_arr+=("Update script: update.sh")
+cat > ./update.sh <<EOF
+$DOCKER_COMMAND compose pull
+$DOCKER_COMMAND up -d --force-recreate
```

Comment:
`$DOCKER_COMMAND up` or `$DOCKER_COMMAND compose up` ?

Reply:
Typo, fixed it :)